semgrep/Makefile

```diff
@@ -3,7 +3,7 @@ all:
 clean:
 	rm -rf ./build/
 test:
-	./tests/run-lint-tests.sh
+	PYTHONPATH=. pytest tests/
```

Comment:
Once semgrep is pip installable, I'd prefer to make sure everyone has it installed as editable in their environment instead of setting PYTHONPATH.